src/events/components/DetailView/PictureCard.tsx

```diff
@@ -13,6 +13,7 @@ const PictureCard = ({ image, event_start, event_end, location, company_event, e
   const startTime = DateTime.fromISO(event_start).toFormat('HH:mm');
   const endDate = DateTime.fromISO(event_end).toFormat('d MMM');
```

Comment:
I would like to remind you that we should follow the official guidance on how to write dates, according to [Språkrådet](https://www.sprakradet.no/sprakhjelp/Skriveregler/Dato/)

Reply:
Good catch, added a fix to this in the latest commit.